com.unity.shadergraph/Documentation~/Internal-Inspector.md

```diff
@@ -0,0 +1,41 @@
+# Graph Inspector
+
+## Description
+The **Graph Inspector**** is used to interact with any graph elements that the user can select and graph-wide settings for the [Shader Graph Asset](Shader-Graph-Asset.md). You can edit their attributes and default values.  
```

Comment:
would change to 
	The **Graph Inspector**** is used to edit the attributes/default values of graph-wide settings [Shader Graph Asset](Shader-Graph-Asset.md), as well as any selectable graph elements.

Reply:
Or something like that, still not great imo